crates/signer/src/signer.rs

```diff
@@ -38,23 +44,46 @@ impl Config {
                 PrivateKeySigner::from_slice(&private_key)
                     .map_err(|_| SignerError::InvalidPrivateKey)
             }
-            Config::Web3(_endpoint, _address) => {
-                todo!() // We are implementing this in a following PR
-            }
         }
     }
```

Comment:
nit: extra line after the previous method

Reply:
Done